internal/controllers/inventory_controller.go

```diff
@@ -1082,6 +1094,69 @@ func (t *reconcilerTask) createAlarmServerClusterRoleBinding(ctx context.Context
 	return nil
 }
 
+// createAlertmanagerClusterRoleAndBinding creates the cluster role and binding needed for the Alertmanager to be able to
+// send alerts to the alarm server.
+func (t *reconcilerTask) createAlertmanagerClusterRoleAndBinding(ctx context.Context) error {
+	name := "alertmanager"
+
+	// Cluster Role
+	role := &rbacv1.ClusterRole{
```

Comment:
Since this is scoped to our namespace we should make this a `Role` instead of a `ClusterRole`

Reply:
All examples I've found use clusterrole and clusterrolebinding. Non resource endpoints are not namescoped.
The official k8s rbac docs has:

Allow GET and POST requests to the non-resource endpoint /healthz and all subpaths (must be in a ClusterRole bound with a ClusterRoleBinding to be effective):